extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -165,6 +187,40 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
       addExperimentIdToElement(verifierEid, this.element);
     }
 
+    const correlator = getCorrelator(this.win);
+    this.ampAnalyticsPageLoadMetricsConfig_ = dict({
+      'requests': {
+        'fvt': 'https://csi.gstatic.com/csi?s=a4a' +
+            `&c=${correlator}` +
+            '&firstVisibleTime.${firstVisibleTime}',
+      },
+      'transport': {
+        'beacon': false,
+        'xhrpost': false,
+      },
+      'triggers': {
+        'iniLoad': {
+          'on': 'visible',
+          'request': 'fvt',
+          'selector': 'amp-ad',
```

Comment:
I'm not precisely sure the semantics of first visible time when applied to a selector pointed at the closest `amp-ad`.  But I think the goal is to get the FVT of the _page_, not of a single _ad slot_.  We're just using the ad slot as a place where we can insert monitoring code so that we can get page-level metrics.  So your selector should probably be pointed more at the whole doc than the slot.

Reply:
Fixed